src/AMSlib/ml/surrogate.hpp

```diff
@@ -393,6 +393,23 @@ class SurrogateModel
   }
 
   bool is_DeltaUQ() { return _is_DeltaUQ; }
+
+  void update(std::string new_path)
```

Comment:
reference?

Reply:
You are right. I Will fix this on a later iteration. Thought you will not review, and @lpottier  did not catch this and I merged once he landed his review.

Reply:
Missed this one indeed (albeit the performance difference should be negligible)